fairlearn/_input_validation.py

```diff
@@ -86,28 +107,37 @@ def _validate_and_reformat_input(X, y=None, expect_y=True, enforce_binary_labels
     else:
         result_y = pd.Series(dtype="float64")
 
-    return pd.DataFrame(X), result_y, pd.Series(sensitive_features.squeeze())
+    return pd.DataFrame(X), result_y, pd.Series(sensitive_features.squeeze()), control_features
```

Comment:
Do we want to either do `squeeze` for both `sensitive_features` and `control_features` here, or both earlier?

Reply:
I don't think it really matters. I think there are larger issues with this function anyway.

Reply:
More of a consistency thing, but I'm not going to block on this.